content/09.overfitting.md

```diff
@@ -6,8 +6,9 @@ In other words, the model fits patterns that are overly specific to the data it
 This subtle distinction is made clearer by seeing what happens when a model is tested on data to which it was not exposed during training: just as a student who memorizes exam materials struggles to correctly answer questions for which they have not studied, a machine learning model that has overfit to its training data will perform poorly on unseen test data.
 Deep learning models are particularly susceptible to overfitting due to their relatively large number of parameters and associated representational capacity.
 Just as some students may have greater potential for memorization, deep learning models seem more prone to overfitting than machine learning models with fewer parameters.
+However, having a large number of parameters does not always imply that a neural network will overfit [@doi:10.1073/pnas.1903070116].
 
-![A visual example of overfitting and failure to generalize. While a high-degree polynomial achieves high accuracy on its training data, it performs poorly on data with specificities that have not been seen before. That is, the model has learned the training dataset specifically rather than learning a generalizable pattern that represents data of this type. In contrast, a simple linear regression works well on both datasets. The greater representational capacity of the polynomial is analogous to using a larger or deeper neural network.](images/overfitting.png){#fig:overfitting-fig}
+![A visual example of overfitting and failure to generalize. While a high-degree polynomial achieves high accuracy on its training data, it performs poorly on the test data that have not been seen before. That is, the model has memorized the training dataset specifically rather than learning a generalizable pattern that represents data of this type. In contrast, a simple linear regression works well on both datasets.](images/overfitting.png){#fig:overfitting-fig}
```

Comment:
>  In contrast, a simple linear regression works well on both datasets.

"well" is relative. The overfitting model performs well on the training set and not well on the test set. The performance of the linear model on both datasets falls between the aforementioned training and test performance. 

Maybe we should change

> "regression works equally well on both datasets."

instead of 

> regression works well on both datasets.

I.e., focusing more on the different between training and test performance. 

I don't feel strongly about that though, because I think it should be obvious what the figure aims to convey.